README.md

```diff
@@ -26,16 +26,28 @@ We currently depend on mavlink C library which is generated by the build
 system during compilation. The corresponding submodule should be fetched.
 
     $ git submodule update --init --recursive
+    
+The following libraries need also to be installed :
+
+    $ sudo apt install python-future
+    $ sudo apt install python3-future
+    $ sudo apt install libtool
+    $ sudo apt install autoconf
```

Comment:
My problem with package names like these is that they change name from version to version. And it is exclusive to one Linux distro. I myself don't use Ubuntu and don't have apt. So I think this could be acceptable by being explicit, starting with "In Ubuntu".

I'm surprised we need both python-future and python3-future.

There are also other dependencies that you probably already had installed, but I think build-essential in Ubuntu should bring everything needed (including gcc). So I think this is the final result:

```
$ sudo apt-get install python-future python3-future build-essential
```

Reply:
I've tried that line using a totally fresh install, and I could not get it to work (I tried it repeatedly with slightly different sequences). I always need to install both autoconf and libtool. When I install build-essential I get the info that the currently installed build-install is already up to date. So, things seem to be consistent.

Reply:
Ok, thanks. Maybe this changed in recent years and they don't include libtool/autoconf anymore. Thanks

Reply:
so, what should I do, consider this conversation as resolved? break the descriptions out into a separate RPi section? Some advice would be much appreciated